internal/rbd/rbd_attach.go

```diff
@@ -27,6 +27,7 @@ import (
 
 	"github.com/ceph/ceph-csi/pkg/util/kernel"
 
+	"github.com/ceph/ceph-csi/internal/csi-common"
```

Comment:
golangci-lint fails on this, but works:
```go
csicommon "github.com/ceph/ceph-csi/internal/csi-common"
```
Will post that once an e2e run has finished.

Reply:
The commit that introduces this change isn't needed. The `--read-only` option is already passed to `rbd map` elsewhere.